compiler/fbs/Converter.hx

```diff
@@ -71,18 +68,17 @@ class Converter {
 		});
 
 		var printer:haxe.macro.Printer = new haxe.macro.Printer();
-		currentModule.types.map(function(t) {
+        for (t in currentModule.types){
 			trace(printer.printTypeDefinition(t));
-		});
+		};
 
 		return currentModule;
 	}
 
 	function storeDeclTypes(parsedObj:ParsedObject):Void {
 		for(field in Reflect.fields(parsedObj)) {
-			Reflect.field(parsedObj, field);
-			(cast Reflect.field(parsedObj, field):Array<Dynamic>).map(function(decl:FbsDeclaration) {
-				switch decl {
+            for (decl in cast ( Reflect.field(parsedObj, field), Array<Dynamic>)) {
```

Comment:
The Lambda.map functions changed (to require a return).  This expression works fine as a for-loop.